server/pbench/bin/dashboard_api_server/create_dashboard.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+import os
+import json
+import sqlite3
+import random
+import configparser
+from datetime import datetime, timedelta
+
+
+def tstos(ts_beg=None, ts_end=None, current=False):
+    """
+    receives list of index names and 
```

Comment:
And it seems like this might be somewhere else in pbench, perhaps we can merge the functionality, or at least keep it with its cousin?